src/back/TicketManager/src/Core/TicketManager.Core.Services/Migrations/20230425162419_AddStatusForEvents.cs

```diff
@@ -0,0 +1,149 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace TicketManager.Core.Services.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddStatusForEvents : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterColumn<byte[]>(
+                name: "RowVersion",
+                table: "Users",
+                type: "bytea",
+                rowVersion: true,
+                nullable: false,
+                defaultValue: new byte[0],
+                oldClrType: typeof(byte[]),
+                oldType: "bytea",
+                oldRowVersion: true,
+                oldDefaultValue: new byte[0]);
+
+            migrationBuilder.AlterColumn<byte[]>(
+                name: "RowVersion",
+                table: "Sectors",
+                type: "bytea",
+                rowVersion: true,
+                nullable: false,
+                defaultValue: new byte[0],
+                oldClrType: typeof(byte[]),
+                oldType: "bytea",
+                oldRowVersion: true,
+                oldDefaultValue: new byte[0]);
+
+            migrationBuilder.AlterColumn<byte[]>(
+                name: "RowVersion",
+                table: "Organizers",
+                type: "bytea",
+                rowVersion: true,
+                nullable: false,
+                defaultValue: new byte[0],
+                oldClrType: typeof(byte[]),
+                oldType: "bytea",
+                oldRowVersion: true,
+                oldDefaultValue: new byte[0]);
+
+            migrationBuilder.AlterColumn<byte[]>(
+                name: "RowVersion",
+                table: "Events",
+                type: "bytea",
+                rowVersion: true,
+                nullable: false,
+                defaultValue: new byte[0],
+                oldClrType: typeof(byte[]),
+                oldType: "bytea",
+                oldRowVersion: true,
+                oldDefaultValue: new byte[0]);
+
+            migrationBuilder.AddColumn<int>(
+                name: "Status",
+                table: "Events",
+                type: "integer",
+                nullable: false,
+                defaultValue: 0);
+
+            migrationBuilder.AlterColumn<byte[]>(
+                name: "RowVersion",
+                table: "Accounts",
+                type: "bytea",
+                rowVersion: true,
+                nullable: false,
+                defaultValue: new byte[0],
+                oldClrType: typeof(byte[]),
+                oldType: "bytea",
+                oldRowVersion: true,
+                oldDefaultValue: new byte[0]);
```

Comment:
Wasn't it your comment to delete this trash from migrations?

Reply:
Yeah, totally forgot about those